### DISCUSSION TITLE
RangeSlider customization

### DISCUSSION THREAD

**Discussion by abdullaev388** [rsuite/rsuite#2623] [Q&A] [answered]
Hello. I am using RangeSlider and i want to have something like this:
If i have range from 0 to 20000 and user slides to 20000 i would like to show infinity symbol instead of 20000.

**Comment by simonguo** [comment, marked as answer]
```tsx
<RangeSlider
  min={0}
  max={20000}
  renderTooltip={value => {
    return value === 20000 ? 'symbol' : value;
  }}
/>
```
